src/extensions/behavior/CommandMenu/component.tsx

```diff
@@ -76,17 +79,29 @@ export const CommandMenuComponent: React.FC<CommandMenuComponentProps> = ({
 function renderItem({id, title, icon, hotkey, hint}: CommandMenuItem): React.ReactNode {
     const titleText = isFunction(title) ? title() : title;
     const hintText = isFunction(hint) ? hint() : hint;
+    const preview = previews[id as keyof typeof ActionName];
```

Comment:
Now the previews are hardcoded in the code, we have to take them from the props, and we should take them from the view item. Add [another attribute](https://github.com/gravity-ui/markdown-editor/blob/main/src/modules/toolbars/types.ts#L22) to the view item and send the preview there, instead of using a single config as it is now.